### PR TITLE
chore(deps): cache CNI/CNS V1.6.0

### PR DESCRIPTION
**Pull request by behzad-mir**
**What type of PR is this?**
/kind feature

Optionally add one or more of the following kinds if applicable:
/kind api-change
/kind deprecation
/kind failing-test
/kind flake
/kind regression
-->

**What this PR does / why we need it**:
caching new CNI/CNS image version V1.6.0
**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:

- [X] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:

```
none
```

### CODE REVIEW

**Review by rbtr — APPROVED** [review]
lgtm, note to AB reviewer that 1.6.x is a new container networking minor release train for AKS 1.30+

File: parts/linux/cloud-init/artifacts/components.json [hunk 1]
```diff
@@ -40,14 +40,21 @@
       "downloadURL": "mcr.microsoft.com/containernetworking/azure-cni:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v1.5.28"
+        "v1.5.28",
+        "v1.6.0"
```
Comment (AbelHu) [inline parts/linux/cloud-init/artifacts/components.json:44]: Do we need to cache this new version on Windows VHDs?

Comment (behzad-mir) [reply parts/linux/cloud-init/artifacts/components.json:44]: Yes, 1.6.0 is only going to be applied on K8s 1.30 and version 1.5.X will still remains active for K8s < 1.30

**Review by AbelHu — APPROVED** [review]
LGTM. Thanks for the PR.

**Pull request merged**